src/Event/EventManager.php

```diff
@@ -461,6 +478,60 @@ public function matchingListeners($eventKeyPattern)
     }
 
     /**
+     * Returns a list of dispatched event objects.
+     *
+     * @return array
+     */
+    public function getDispatchedEvents()
+    {
+        return $this->_dispatchedEvents;
+    }
+
+    /**
+     * Enables the stacking of dispatched events.
+     *
+     * @return void
+     */
+    public function enableEventStacking()
+    {
+        $this->_eventStacking = true;
+    }
+
+    /**
+     * Disables the stacking of dispatched events.
+     *
+     * @return void
+     */
+    public function disableEventStacking()
```

Comment:
Guess it's a matter of interpretation. You build up a stack (to stack) them or you can for sure say as well "I'm tracking" them. I'll change it.